rust/src/lib.rs

```diff
@@ -1201,6 +1239,43 @@ pub extern "C" fn zcashlc_get_sent_memo_as_utf8(
     unwrap_exc_or_null(res)
 }
 
+/// Returns the memo for a sent note, by copying the corresponding bytes to the received
+/// pointer in `memo_bytes_ret`.
+///
+/// The note is identified by its row index in the `sent_notes` table within the data
+/// database.
+///
+/// # Safety
+///
+/// - `db_data` must be non-null and valid for reads for `db_data_len` bytes, and it must have an
+///   alignment of `1`. Its contents must be a utf-8 string representing a valid system path.
```

Comment:
This doc comment is also incorrect. Now that the comment for `zcashlc_get_received_memo` is being fixed in this PR, this one should be as well.